src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -790,30 +795,30 @@ $dark-focusColor: var(--green-500, color(green-500));
   background-color: var(--KCodeBlockBackgroundColor, $light-backgroundColor);
   border-radius: var(--KCodeBlockBorderRadius, $borderRadius);
   display: grid;
-  gap: var(--spacing-sm, spacing(sm));
+  gap: var(--spacing-sm, var(--kui-space-50, $kui-space-50));
   grid-template-columns: var(--maxLineNumberWidth) 1fr;
-  margin-bottom: 0;
-  margin-top: 0;
+  margin-bottom: var(--kui-space-0, $kui-space-0);
+  margin-top: var(--kui-space-0, $kui-space-0);
   max-height: var(--KCodeBlockMaxHeight, none);
   min-height: 56px;
   overflow: auto;
-  padding: var(--spacing-md, spacing(md)) 0 0 var(--spacing-sm, spacing(sm));
+  padding: var(--spacing-md, var(--kui-space-60, $kui-space-60)) var(--kui-space-0, $kui-space-0) var(--kui-space-0, $kui-space-0) var(--spacing-sm, var(--kui-space-50, $kui-space-50));
 }
 
 .k-code-block pre.is-single-line {
   grid-template-columns: auto;
-  padding: var(--spacing-sm, spacing(sm)) var(--spacing-xxl, spacing(xxl)) 0 0;
+  padding: var(--spacing-sm, var(--kui-space-50, $kui-space-50)) var(--spacing-xxl, var(--kui-space-110, $kui-space-110)) var(--kui-space-0, $kui-space-0) var(--kui-space-0, $kui-space-0);
 
   code {
-    line-height: 29px;
-    margin-right: 20px;
+    line-height: var(--kui-line-height-50, $kui-line-height-50);
+    margin-right: var(--kui-space-70, $kui-space-70);
     overflow-x: auto;
-    padding-bottom: var(--spacing-xs, spacing(xs));
-    padding-left: var(--spacing-sm, spacing(sm));
+    padding-bottom: var(--spacing-xs, var(--kui-space-40, $kui-space-40));
+    padding-left: var(--spacing-sm, var(--kui-space-50, $kui-space-50));
   }
 
   + .k-code-block-copy-button {
-    top: var(--spacing-xs, spacing(xs));
+    top: var(--spacing-xs, 8px);
```

Comment:
**Suggestion**: Should this use the same fallback style instead of a fixed pixel value?

```suggestion
    top: var(--spacing-xs, var(--kui-space-40, $kui-space-40));
```

Reply:
No, we don't use space tokens for positioning properties. Those properties should just be pixel values for now (keeping the old variables where applicable)